src/cpp/fastdds/subscriber/Subscriber.cpp

```diff
@@ -55,7 +55,9 @@ ReturnCode_t Subscriber::enable()
     }
 
     enable_ = true;
```

Comment:
This is not needed anymore

Reply:
Yes, it is. Inside `impl_->enable()`, we may call the `enable()` method on previously created readers, which will fail if the subscriber is not enabled.